docs/ums/README.md

```diff
@@ -153,12 +224,26 @@ You can do user management via CRUD operations. This method is mainly for the cu
 ::: warning
 This will be deprecated in future versions.
 :::
-<CodeSwitcher :languages="{js:'JavaScript',bash:'cURL'}">
+<CodeSwitcher :languages="{js:'JavaScript',py:'Python',bash:'cURL'}">
 <template v-slot:js>
 
 ``` js
-ums.deleteUser('Elon@tesla.com', password); 
+ums.deleteUser('Elon@tesla.com', password)
+.subscribe(user => {}, error=>{});    
+```
+</template>
+<template v-slot:py>
+
+``` py
+...
```

Comment:
I would suggest to use full example of code everytime, because some users can open this part of documentation by directly link. And in this case they need to find another part of code.

Reply:
next version